README.md

```diff
@@ -228,7 +211,7 @@ Also known as the `client_id`. You must additionally pass in your Adobe API key
 
 ## Rate Limiting
 
-We have not put a throttle limit on requests to the API at this time.
+All free trial users will have 5,000 API calls in order to test and evaluate the API in a non production environment. If for any reason you reach your limit and need to extend your quota please reach out psdservices@adobe.com for more information. 
```

Comment:
@archyposada Throttling(Rate limiting) and Trial limit are two different things. Let's keep the previous line . May be something like this ? `We have not put a throttle limit on requests to the API at this time.
However we limit Free Trials. All free trial users will have 5,000 API calls in order to test and evaluate the API in a non production environment. If for any reason you reach your limit and need to extend your quota please reach out psdservices@adobe.com for more information`

Reply:
Why don't we leave the rate limiting section the same with the old sentence and add new section labeled Quota Limits and add:
"All free trial users will have 5,000 API calls in order to test and evaluate the API in a non production environment. If for any reason you reach your limit and need to extend your quota please reach out psdservices@adobe.com for more information. "

Reply:
@archyposada That sounds good to me.